exporters/memory/test/in_memory_span_data_test.cc

```diff
@@ -0,0 +1,33 @@
+#include "opentelemetry/exporters/memory/in_memory_span_data.h"
+#include "opentelemetry/nostd/span.h"
+#include "opentelemetry/sdk/trace/span_data.h"
+
+#include <gtest/gtest.h>
+
+using opentelemetry::exporter::memory::InMemorySpanData;
+using opentelemetry::sdk::trace::Recordable;
+using opentelemetry::sdk::trace::SpanData;
+
+TEST(InMemorySpanData, AddRecordable)
+{
+  InMemorySpanData data(100);
+
+  ASSERT_EQ(0, data.GetSpans().size());
+
+  opentelemetry::nostd::span<std::unique_ptr<Recordable>> recordables(
+      new std::unique_ptr<Recordable>(new SpanData), 1);
```

Comment:
You should be able to just do std::make_unique\<SpanData\>(). Since Recordable is a parent of SpanData, the cast should be implicit.

Reply:
make_unique was added in C++ 14, and we're running on C++ 11, so this won't be possible